### PR TITLE
Fix crashing WebUI identity tests in Nightly PR

### PR DESCRIPTION
**Pull request by serg-cymbaluk**
Clear default user/host group before deleting it in "Automember" tests.

### CODE REVIEW

File: ipatests/prci_definitions/gating.yaml [hunk 1]
```diff
@@ -3,14 +3,10 @@ topologies:
     name: build
     cpu: 2
     memory: 3800
-  master_1repl: &master_1repl
```
Comment (tiran) [inline ipatests/prci_definitions/gating.yaml]: Tip: We have a special template for temp commits, which is easier to use. You can activate the template with a single command:

``ln -sf ipatests/prci_definitions/temp_commit.yaml .freeipa-pr-ci.yaml``

Comment (serg-cymbaluk) [reply ipatests/prci_definitions/gating.yaml]: Ok, thank you

**Comment by serg-cymbaluk** [comment]
@tiran 
The PR is rebased, checks have passed.

**Comment by serg-cymbaluk** [comment]
master:

* 07c163ca9214b1d6bb2446dd3c93ad04e1382535 Fix "ID views" tests fail after running "Automember" tests

**Pull request closed without merging**